docs/schemas/resource/IfcGeometricConstraintResource/Entities/IfcLinearPlacement.md

```diff
@@ -5,7 +5,8 @@ _IfcLinearPlacement_ provides a specialization of _IfcObjectPlacement_ in which
 ## Attributes
```

Comment:
Some comments, I've encapsualated them into the suggestions below:

### Re `RelativePlacement`

Resemblance of equivalent attributes on other placement types (At least the word placement should probably be used)

**grid**: Placement of the object coordinate system defined by the intersection of two grid axes.

**local**: Geometric placement that defines the transformation from the related coordinate system into the relating. The placement can be either 2D or 3D, depending on the dimension count of the coordinate system.

### Re  `CartesianPosition`

Reverse the order of the statements

Make it unambiguous that this is only the equivalent of the relative part, not the including the absolute transform. 

Mention *importing* applications can use it. Metnion optional.

### Re 2d / 3d

For IfcPointOnCurve we have "The coordinate space dimensionality of the point is that of the basis curve.". That is very clear and elegant. IfcPointByDistanceExpr is also has dimensionality derived from its basis (IfcPointDim), so can be 2d. When we say linear placement is by definition 3d I don't find it very elegant (did nobody consider 2d objects contexts along a horizontal alignment?). But it's in the entity text and the cartesian fallback is also 3d specifically, so no point in arguing this. but if a placement along a 2d curve is 3d in nature I think it deserves some additional emphasis [also created #846].